test-requirements.txt

```diff
@@ -0,0 +1 @@
+mock
```

Comment:
I don't understand this change. `mock` is part of `unittest` now...

Reply:
Pipeline was failing: https://github.com/OCA/storage/actions/runs/4163534197/jobs/7204052623

Reply:
Because the import is wrong :wink: